client/src/app/project/project.component.html

```diff
@@ -31,6 +33,15 @@
       </thead>
       <tbody>
       <tr *ngFor="let submissionEnvelope of submissionEnvelopes">
+        <td>
+          <mat-icon [ngStyle]="{'color': 'lightgrey'}" *ngIf="submissionEnvelope['isUpdate']"
```

Comment:
Can just use `style` when it's not a dynamic style right?

Reply:
yeah but most of the codes using mat-icon in this repo are actually similar to this. We could change all those in a separate PR.